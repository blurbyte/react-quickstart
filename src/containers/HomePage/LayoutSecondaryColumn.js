import styled from 'styled-components';
import media from '../../styles/media';

const LayoutSecondaryColumn = styled.div`
  width: 30%;
  ${media.desktop`width: 46%;`} ${media.tablet`width: 100%;`};
`;

export default LayoutSecondaryColumn;
