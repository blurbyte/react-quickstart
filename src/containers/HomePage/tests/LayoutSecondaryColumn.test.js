import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import LayoutSecondaryColumn from '../LayoutSecondaryColumn';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(<LayoutSecondaryColumn />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
