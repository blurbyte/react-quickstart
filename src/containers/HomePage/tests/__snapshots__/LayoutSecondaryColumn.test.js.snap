// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.c0 {
  width: 30%;
}

@media (max-width:1000px) {
  .c0 {
    width: 46%;
  }
}

@media (max-width:700px) {
  .c0 {
    width: 100%;
  }
}

<div
  className="c0"
/>
`;
